app.py

```diff
@@ -88,33 +88,58 @@ def return_code(self):
         return self._return_code
 
 
-@app.route('/execute', methods=['POST'])
-def execute():
+def _execute(request_as_string: str):
     success = False
-    req = request.get_json()
     try:
         conn = get_conn_through_sentinel()
-        response = conn.execute_command(*req['command'].split())
+        response = conn.execute_command(*request_as_string.split())
         success = True
     except (redis.exceptions.ConnectionError, redis.exceptions.ResponseError):
         try:
             reload_username_password_from_file_system_if_needed(app)
             conn = get_conn_through_sentinel()
-            response = conn.execute_command(*req['command'].split())
+            response = conn.execute_command(*request_as_string.split())
             success = True
         except Exception as err:
             response = 'Exception: cannot connect. %s' % str(err)
             app.logger.exception("execute err")
     except Exception as err:
         response = 'Exception: %s' % str(err)
         app.logger.exception("execute err")
+    return response, success
```

Comment:
It might be a good idea to change the return value of success to an error message as a string. In this case, an empty string would indicate success, much like the way Golang handles errors.

Note: by doing so, you will need to fix the web_cli.

https://github.com/RedisLabs/redis-enterprise-operator/blob/main/integration_tests/src/webcli/redis_web_cli_client.py#L52

Reply:
good idea and I agree, but out of scope of this ticket, especially since there are other calls that already follow this convention.